modules/nested-tabs/widgets/nested-tabs.php

```diff
@@ -81,7 +81,7 @@ protected function register_controls() {
 		$tooltip_start = is_rtl() ? esc_html__( 'Right', 'elementor' ) : esc_html__( 'Left', 'elementor' );
 		$tooltip_end = is_rtl() ? esc_html__( 'Left', 'elementor' ) : esc_html__( 'Right', 'elementor' );
 		$nested_tabs_heading_selector_class = ':is( {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-heading, {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content )';
-		$nested_tabs_content_selector_class = ':where( {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content ) > .e-con';
+		$nested_tabs_content_selector_class = ':where( {{WRAPPER}} > .elementor-widget-container > .e-n-tabs > .e-n-tabs-content ) > .e-con.e-active';
```

Comment:
I think that using the `.e-con.e-active` as the selector to target for the many styling controls will have more adverse effect than helpful effects.
If a css `transition: 1s;` (for example) is added for any reason (by a theme, by a future control, by a user, by a plugin that adds transision features to the container widget, etc), then then what you will immediately see  when the tab is shown you will also for example see the `border-color` transition from the default `transparent, black, white` (whatever it is) to your chosen color, and the same for any styled properties `border-width`, `padding`, even `background-color`, and etc. So I this selector should stay as it is.

You and I have spoekn about this in a huddle, I've noted my feedback here, and you already have another JS option, which I like.

We can invlove Udi in the AM and chose the best option.

Nice work.